src/components/Modal/ApplyForm.js

```diff
@@ -4,149 +4,220 @@ import {
-                                required
-                                min="0"
-                                name="Mobile"
-                                id="Phone_Number"
-                                type="text"
-                                placeholder="+918080808080"
-                                minLength={10}
-                                maxLength={13}
-                                onChange={this.handleChange}
-                            />
-                        </FormGroup>
-                    </Col>
-                </Row>
-                <Row>
-                    <Col >
-                        <h5>Email ID</h5>
-                        <FormGroup className="mt-0 pt-0">
-                            <Input
-                                required
-                                name="Email"
-                                id="Email"
-                                type="email"
-                                placeholder="name@iitk.ac.in"
-                                pattern=".+@iitk\.ac\.in"
-                                onChange={this.handleChange}
-                            />
-                        </FormGroup>
-                    </Col>
-                </Row>
-                <h3 className="mt-3">Additional Information</h3>
-                <hr />
-                <Row>
-                    <Col lg="6">
-                        <h5>Linkedin Profile</h5>
-                        <FormGroup>
-                            <Input
-                                required
-                                name="LinkedIn"
-                                id="Linkedin"
-                                type="url"
-                                placeholder="https://linkedin.com/in/name"
-                                onChange={this.handleChange}
-                            />
-                        </FormGroup>
-                    </Col>
-                    <Col lg="6">
-                        <h5>Github Profile</h5>
-                        <FormGroup>
-                            <Input
-                                required
-                                id="Github"
-                                type="url"
-                                placeholder="https://github.com/name"
-                                // pattern='https:\/\/github\.com\/[^\/]+\/'
-                                onChange={this.handleChange}
-                            />
-                        </FormGroup>
-                    </Col>
-                </Row>
-                <hr style={{ margin: '12px 0' }} />
-                <Row className='text-right'>
-                    <Col>
-                        <Button type='submit' color="success">Submit</Button>
-                        <Button outline color="secondary" onClick={this.props.toggle}>Cancel</Button>
-                    </Col>
-                </Row>
-            </Form>
-        )
+        if (this.state.data.fetched){
```

Comment:
instead of if-else you can use something like {this.state.data.fetched && ()} or you can use the ternary operator here if you want else inside of return.
And you can define all state functions early that will const {any values} = this.state this way